Networking/Networking.xcodeproj/project.pbxproj

```diff
@@ -245,6 +245,11 @@
 		93D8BBFD226BBEE800AD2EB3 /* AccountSettingsMapper.swift in Sources */ = {isa = PBXBuildFile; fileRef = 93D8BBFC226BBEE800AD2EB3 /* AccountSettingsMapper.swift */; };
 		93D8BBFF226BC1DA00AD2EB3 /* me-settings.json in Resources */ = {isa = PBXBuildFile; fileRef = 93D8BBFE226BC1DA00AD2EB3 /* me-settings.json */; };
 		93D8BC01226BC20600AD2EB3 /* AccountSettingsRemoteTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 93D8BC00226BC20600AD2EB3 /* AccountSettingsRemoteTests.swift */; };
+		AE0C144D258BC4E80023C0F8 /* CustomerMapper.swift in Sources */ = {isa = PBXBuildFile; fileRef = AE0C144C258BC4E80023C0F8 /* CustomerMapper.swift */; };
+		AE0C1451258BCA0E0023C0F8 /* CustomerMapperTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = AE0C1450258BCA0E0023C0F8 /* CustomerMapperTests.swift */; };
+		AE0C1457258BCB700023C0F8 /* customer.json in Resources */ = {isa = PBXBuildFile; fileRef = AE0C1454258BCAA50023C0F8 /* customer.json */; };
+		AE0C149B258BCB7A0023C0F8 /* customers-extra.json in Resources */ = {isa = PBXBuildFile; fileRef = AE9CEC78258913D5000F3CAB /* customers-extra.json */; };
+		AE0C149C258BCB7D0023C0F8 /* customers-empty.json in Resources */ = {isa = PBXBuildFile; fileRef = AE9CEC79258913D8000F3CAB /* customers-empty.json */; };
```

Comment:
It's curious that Xcode shows these changes. In theory, these files were added in a previous PR, right?

Reply:
They weren't added to test target but worked anyway. Since everything else is added, I've added them too.

<img width="173" alt="image" src="https://user-images.githubusercontent.com/3132438/102607828-04d66480-413a-11eb-8ca2-0c11452733ce.png">